lib/child_process.js

```diff
@@ -207,7 +205,11 @@ exports.execFile = function(file /*, args, options, callback*/) {
     }
 
     ex.cmd = cmd;
-    callback(ex, stdout, stderr);
+    if (!callback){
```

Comment:
This line fails linting. (On Linux/Mac, `make jslint`.)  Needs a spec before the curly brace.